enzyme/tools/enzyme-tblgen/enzyme-tblgen.cpp

```diff
@@ -1203,9 +1166,27 @@ void handleUse(
   bool usesShadow = Def->getValueAsBit("usesShadow");
   bool usesCustom = Def->getValueAsBit("usesCustom");
 
-  // We don't handle any custom primal/shadow
-  (void)usesCustom;
-  assert(!usesCustom);
+  // This only concerns instances of StaticSelect for now
+  if (usesCustom) {
```

Comment:
very minor but can we explicitly check opName == "StaticSelect" || Def->isSubClassOf("StaticSelect") and leave this assertion as is?